run-tests.sh

```diff
@@ -12,8 +12,13 @@
 # removed when https://github.com/pyupio/safety-db/pull/2274 is merged and
 # released.
 
-docker-services-cli up postgresql es redis
-python -m check_manifest --ignore ".travis-*" && \
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
```

Comment:
Could `set -e` be enough? Why was this added now?

Reply:
`set -e` would replace the first, `set -u` would replace the second. I was just trying to follow what was agreed on expanding command-line arguments to be more explicit.

Why? Because it was not enough as it was. For example, if the manifest check was wrong it would continue to run tests. We want it to fail fast and stop in the first command that errors.

Reply:
ok!